src/Entity/ProgrammaticFoundation/Measure.php

```diff
@@ -0,0 +1,290 @@
+<?php
+
+namespace AppBundle\Entity\ProgrammaticFoundation;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use AppBundle\Entity\EntityIdentityTrait;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Mapping\Annotation as Gedmo;
+use Gedmo\Timestampable\Traits\TimestampableEntity;
+use Ramsey\Uuid\Uuid;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="programmatic_foundation_measure")
+ * @UniqueEntity("slug", message="programmatic_foundation.slug.unique.measure")
+ * @UniqueEntity(
+ *     fields={"position", "subApproach"},
+ *     errorPath="position",
+ *     message="programmatic_foundation.unique_position.measure"
+ * )
+ * @Algolia\Index(autoIndex=false)
+ */
+class Measure
+{
+    use EntityIdentityTrait;
+    use TimestampableEntity;
+
+    /**
+     * @ORM\Column(type="smallint")
+     * @Assert\GreaterThan(value=0, message="programmatic_foundation.position.greater_than_zero")
+     */
+    private $position;
+
+    /**
+     * @ORM\Column
+     * @Assert\NotBlank(message="programmatic_foundation.title.not_empty")
+     */
+    private $title;
+
+    /**
+     * @ORM\Column(unique=true)
+     * @Gedmo\Slug(fields={"title"})
+     * @Assert\NotBlank(message="programmatic_foundation.slug.not_empty")
+     */
+    private $slug;
+
+    /**
+     * @ORM\Column(type="text")
+     * @Assert\NotBlank(message="programmatic_foundation.content.not_empty")
+     */
+    private $content;
+
+    /**
+     * @ORM\Column(type="boolean", nullable=false)
```

Comment:
on this project we don't specify default values. and `nullable=false` is the default value of the `Column` annotation.